README.md

```diff
@@ -0,0 +1,55 @@
+# PaaSTech Proto Definitions
+
+Welcome to the PaaSTech Proto Definitions repository!
+
+This repository contains the protocol buffer definitions for PaaSTech Cloud services.
+
+This README provides instructions on how to use the proto definitions and how to add new ones.
+
+## Usage
+
+Using the PaaSTech Proto Definitions is straightforward.
+
+Just follow these simple steps:
+
+1. Open your `Cargo.toml` file in your project.
+2. Under the `[dependencies]` section, add the following line:
+
+```toml
+[dependencies]
+# ...
+paastech-proto = { git = "https://github.com/paastech-cloud/proto.git", branch = "main" }
+```
+
+Alternatively, you can specify a specific version using a commit hash:
+
+```toml
+[dependencies]
+# ...
+paastech-proto = { git = "https://github.com/paastech-cloud/proto.git", rev = "1520c87" }
```

Comment:
You could explain the implications of using either rev or branch, especially concerning breaking changes

Reply:
It's the preference of the developer, I don't think we need to precise it.